aptos-move/move-examples/fungible_asset/stablecoin/sources/fiat_token.move

```diff
@@ -0,0 +1,420 @@
+module circle::fiat_token {
+    use aptos_framework::fungible_asset::{Self, MintRef, TransferRef, BurnRef, Metadata};
+    use aptos_framework::object::{Self, Object};
+    use aptos_framework::primary_fungible_store;
+    use std::error;
+    use std::string::{Self, String};
+    use std::option;
+    use std::signer;
+    use std::vector;
+    use aptos_std::smart_table::{Self, SmartTable};
+
+    friend circle::master_minter;
+
+    /// Error codes documentation: https://aptos.dev/reference/error-codes/
+    const EALREADY_INITIALIZED: u64 = 1;
```

Comment:
Can you add a error comment for each of these errors? Error comments are used as human-readable error messages returned by APIs